src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
-server.port=4056
+server.port=${EXTENSIONS_API_PORT}
```

Comment:
seeing as it was defaulting to port 4056, could we try using ${EXTENSIONS_API_PORT:4056} instead, that way it will use the provided port if available and revert back to 4056 if not.

Reply:
Added default.